### ISSUE TITLE
More appropriate colors

### ISSUE THREAD

**Issue by aemogie** [catppuccin/windows-terminal#15]
While in Latte the contrast is fine, for Mocha, Macchiato and Frappe it is very low.
The colours used are
<table>
<tr>
  <th>Flavour</th>
  <th>Surface 1</th>
  <th>Surface 2</th>
</tr>
<tr>
  <td>🌿 Mocha</td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/mocha_surface1.png" height="23" width="23"/></td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/mocha_surface2.png" height="23" width="23"/></td>
</tr>
<tr>
  <td>🌺 Macchiato</td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/macchiato_surface1.png" height="23" width="23"/></td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/macchiato_surface2.png" height="23" width="23"/></td>
</tr>
<tr>
  <td>🪴 Frappé</td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/frappe_surface1.png" height="23" width="23"/></td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/frappe_surface2.png" height="23" width="23"/></td>
</tr>
</table>

For reference, the default "Campbell" theme uses `#0C0C0C` for dark black, and `#767676` for the bright variant. If we were to find a colour pair with similar darkness in the palette it would be Mantle and Overlay 0, respectively.

<table>
<tr>
  <th>Flavour</th>
  <th>Mantle</th>
  <th>Overlay 0</th>
</tr>
<tr>
  <td>🌿 Mocha</td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/mocha_mantle.png" height="23" width="23"/></td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/mocha_overlay0.png" height="23" width="23"/></td>
</tr>
<tr>
  <td>🌺 Macchiato</td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/macchiato_mantle.png" height="23" width="23"/></td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/macchiato_overlay0.png" height="23" width="23"/></td>
</tr>
<tr>
  <td>🪴 Frappé</td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/frappe_mantle.png" height="23" width="23"/></td>
  <td><img src="https://github.com/catppuccin/catppuccin/raw/main/assets/palette/circles/frappe_overlay0.png" height="23" width="23"/></td>
</tr>
</table>

In addition it should be mentioned that, the "Bright White" is _darker_ than the "Dark White".

**Comment by aemogie** [comment]
I noticed that some of the other colours could be changed as well, so I changed the title.
I experienced these on Mocha, but I assume it's the same for the other palettes as well.

1. The dark purple and bright purple colours, use the _Pink_ from the colour palette, over the more appropriate _Mauve_. Maybe change the dark purple to _Mauve_, and leave the bright purple using _Pink_?
2. The dark variant for yellow could be possibly changed to use _Peach_. It has a close enough hue, and the value is darker than the _Yellow_ from the palette. The bring variant will continue to use _Yellow_.
3. There's 4 blues in the colour palette (excluding _Teal_, which is reserved for cyan ANSI colour) - _Sky_, _Sapphire_, _Blue_ and _Lavender_. So 2 of these can be picked for the 2 blue ANSI colours.
